pages/index.tsx

```diff
@@ -4,7 +4,7 @@ import Clients from '../components/Clients'
 import Services from '../components/Services'
 import constants from '../data/constants.json'
 
-export default (props: {}) => (
+export default () => (
```

Comment:
FYI you don't need those braces `()` its only when you return an object `{}` its required